website/docs/architecture.md

```diff
@@ -4,20 +4,20 @@ sidebar_position: 2
 
 # Architecture
 
-Open Services Cloud is a framework reshaping the cloud service ecosystem:
+xpanse is a framework reshaping the cloud service ecosystem:
 
 * the cloud users can find the same services across different cloud providers, exactly the same services provided by
-  Open Services Cloud.
+  xpanse.
 * the software vendors can easily create native and portable managed services for their software, including seamless and
   low level integration with cloud provider services. In addition of software artifacts, software vendor describe the
-  service using the Open Services Cloud Configuration Language (OCL).
+  service using the xpanse Configuration Language (OCL).
```

Comment:
OCL must be still used as it is.

Reply:
Yes, we have discussed this, but I forgot to push it up.
BYW, Should we only rename the "demo" folder to "mock-ui", but leave the contents as it is?

Reply:
yes, only the folder must be renamed.